include/data_management/data/soa_numeric_table.h

```diff
@@ -447,7 +447,6 @@ class DAAL_EXPORT SOANumericTable : public NumericTable
         return services::Status();
     }
 
-private:
     services::Status generatesOffsets()
```

Comment:
why it became `public` method?

Reply:
protected, I need to call this method in the heir